avm/ptn/sa/content-processing/main.bicep

```diff
@@ -0,0 +1,1562 @@
+// ========== main.bicep ========== //
```

Comment:
Please make sure the value of `enableTelemetry` is passed through to all files that deploy AVM modules as a user must be able to disable telemetry via this one simple parameter